slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -0,0 +1,343 @@
+> **- Matt Jago, Ops & infrastructure**
+:::
+::::
+
+:::: {.column width="33%"}
+::: {.fragment .fade-up}
+> *"\[The improvements have\] reduced the human errors and also mean analysts and LA colleagues’ time can be spent on more valuable tasks."*\
+> **- Family hubs research and analysis team**
+:::
+::::
+
+:::: {.column width="33%"}
+::: {.fragment .fade-up}
+> *"The script itself is not complicated, but it removes the risk of human error and reduces the production time to minutes. The time saved can then be dedicated to further quality assurance."*\
+> **- RAAC response analyst**
+:::
+::::
+:::::::::
+
+# Improving pipeline efficiency: before RAP
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="80%"}
+![](images/RAP-flowchart-white-before.png)
+:::
+
+::: {.column width="10%"}
+:::
+
+# Improving pipeline efficiency: after RAP
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="80%"}
+![](images/RAP-flowchart-white-after.png)
+:::
+
+::: {.column width="10%"}
+:::
+
+# What it means for managers
+
+On our [RAP expectations page](https://dfe-analytical-services.github.io/analysts-guide/RAP/rap-expectations.html#analyst-managers), we have pulled out the expectations for analyst managers from the [cross-government RAP strategy](https://analysisfunction.civilservice.gov.uk/policy-store/reproducible-analytical-pipelines-strategy/), which include:
+
+> -   build extra time into projects to adopt new skills and practices where appropriate
+> -   learn the skills they need to manage software
+> -   evaluate RAP projects within organisations to understand and demonstrate the benefits of RAP
+> -   mandate their teams use RAP principles whenever possible
+
+# What does it actually mean for managers?
+
+# A nightmare scenario...
+
+-   You get a call from your team member, they're too ill to come to work this week
+-   They were the only one who knew how to run the code for your publication
+-   You're already behind schedule
+
+No problem! You can just run the code yourself, right?
+
+# Ok, first I need to find the code...
+
+::: {.column width="100%"}
+:::
+
+::: {.column width="40%"}
+This should be easy, it's in our shared folder because we haven't had a chance to move it to Azure DevOps yet...
+:::
+
+::: {.column width="60%"}
+![](images/code_nightmare_1.png){fig-align="center" width="60%"}
+:::
+
+# 
+
+::: {.column width="100%"}
+![](images/one-hour-later.jpg){fig-align="center" width="60%"}
+:::
+
+# Ok, I think I've found the right code...
+
+::: {.column width="100%"}
+:::
+
+::: {.column width="40%"}
+It's been a little while since I've run this code, I ought to check the documentation to make sure I do it right
+:::
+
+::: {.column width="60%"}
+![](images/code_nightmare_2.png){fig-align="center" width="75%"}
+:::
+
+# I'm sure there'll be comments in the code that tell me what to do...
+
+::: {.column width="100%"}
+:::
+
+::: {.column width="100%"}
+![](images/code_nightmare_3.png){fig-align="center" width="50%"}
```

Comment:
might be helpful to imply a manual step in this too - like a comment that says #take a copy of the excel table published here and add it to the repo or something similar. 
We don't really mention the manual steps thing through this but I think they're still a big problem?

Reply:
would it be more nightmareish to move above the read_csv bit and add comment like take copy of table 3 from latest [insert publication name here] (like the ONS population estimates (or similar)). 

Like you have no idea what the data actually is, there's no link, and no previous version to work from